app/konnect/runtime-manager/index.md

```diff
@@ -24,13 +24,89 @@ for each runtime instance.
 
 ## Runtime groups
 
-{{site.konnect_short_name}} manages runtime configuration in runtime groups. Each runtime group acts
+{{site.konnect_short_name}} manages runtime configuration in runtime groups. A **runtime group** is a collection of API connectivity runtime instances
+sharing the same configuration and behavior space. Each runtime group acts
 as a separate control plane and can manage runtime configurations independently
-of any other group. See [runtime groups](/konnect/runtime-manager/runtime-groups)
-for more information.
+of any other group. You can find a list of all runtime groups in your organization
+on the [Runtime Manager overview](https://cloud.konghq.com/runtime-manager/).
+
+Every organization has one default runtime group. The default runtime group is the foundational group in {{site.konnect_short_name}}.
+Every region in every organization starts with one default group. This group can't be deleted, and its status as the default group can't be changed.
+
+With an [Enterprise subscription](https://konghq.com/pricing/), you can configure additional
+custom runtime groups. Use multiple groups in one {{site.konnect_short_name}} organization to
+manage runtime instances and their configuration in any groupings you want.
+
+Some common use cases for using multiple runtime groups include:
+
+* **Environment separation:** Split environments based on their purpose, such as
+development, staging, and production.
+* **Region separation:** Dedicate each runtime group to a region or group of
+regions. Spin up runtime instances in those regions for each runtime group.
+* **Team separation:** Dedicate each runtime group to a different team and share
+resources based on team purpose.
+
+![runtime groups](/assets/images/docs/konnect/konnect-runtime-groups-example.png)
+> _**Figure 1:** Example runtime group configuration with three runtime groups: the default group, a development group, and a production group. {{site.konnect_short_name}} is the SaaS-managed global management plane that controls all of the groups, while the runtime groups contain self-managed runtime instances._
+
+Access to each runtime group is configurable on a team-by-team basis using
+entity-specific permissions. For more information, see [Administer teams](/konnect/org-management/teams-and-roles).
+
+[Set up and manage runtime groups &rarr;](/konnect/runtime-manager/manage-runtime-groups/)
```

Comment:
```suggestion
```
let's remove this. The screenshot for the UI is supposed to cover this; we're only going to have a separate runtime groups with decK topic.